### DISCUSSION TITLE
有考虑做兼容移动端嘛  我觉得你们的这个ui库 很好看

### DISCUSSION THREAD

**Discussion by lovesL** [DouyinFE/semi-design#287] [Ideas]
### Which Component Need Enhancement 期望新增功能的组件
- Button

**Comment by boomboomchen** [comment]
响应式和跨端是semi后续的一个重点考虑方向，具体的规划后续会同步给大家哈

**Comment by lovesL** [comment]
> 响应式和跨端是semi后续的一个重点考虑方向，具体的规划后续会同步给大家哈

期待 +1

**Comment by shijiatongxue** [comment]
此问题迁移到 discussion，大家可以去这里讨论与代码无关的话题 ❤

**Comment by Tim-Paik** [comment]
首先很感谢Semi团队的付出

我在手机上看到这个项目的时候第一反应是没做手机端，首页没做，文档也没做、、

不过后来在电脑上仔细看文档的过程还是给了我很大震撼：字节跳动也确确实实在认真做开源了，好看，好用

希望你们能认真做好移动端适配，还有就是建议放几个 Demo 出来让大家看看效果，也许能让更多的人了解到 Semi Design ( 或者我更愿意称其为 Semi UI Kit )

**Comment by royalzzz** [comment]
非常惊艳的前端组件库，个人也在一直关注该项目的持续发展，多次安利我所在工作团队更换这个组件库，非常可惜我们需要做到所有终端的风格样式统一，如果能推出移动端的UI组件库我觉得我们选择Semi就没有任何顾虑的理由了。

👍

**Reply by pointhalo** [reply]
感谢认可。上面的回答间隔比较久了，我重新更新一下近期我们内部对这个事项的看法。

可预见的一两年内，我们应该都不会有多做一个 Semi 移动端组件库的打算。**简单来说，是因为我们自身没有这类诉求。**

Semi 在 19 年立项时，是因为抖音业务的一些中后台场景，对主题 Design Token定制，暗色模式，现代化的设计风格，更易用的 Form 表单组件 API设计等有强诉求，而彼时市面上的 lib 都无法直接满足我们诉求，才决定从0到1自建。一直稳定迭代到现在，离不开内部业务需求的强依赖和支撑。

但是对于移动端而言，我们团队没有这方面相关的诉求。我们业务对于 mobile类场景的需求，大多通过抖音 App 进行了承载，这些业务中，使用的是同样由我们团队开发维护的抖音移动端组件库。但由于它是跟与抖音App 宿主强绑定的，无论是风格、交互、功能，还是代码与iOS/Android native code的耦合，定位跟 Semi 都完全不一样，因此它不适用于开放场景，主要是供我们内部消费。

所以如果 Semi 光做一个适配或者重新再开发一个 mobile 库，目前阶段，无法反哺我们自身的业务，我们自己没法在业务中用起来的话，那只能用爱发电，可投入的研发资源是非常有限的，也没法保证长期投入，无论是迭代的持续性，还是质量稳定性我们都无法保障。我们并不想做一锤子买卖。
